src/Static.jl

```diff
@@ -519,9 +519,9 @@ Base.:(!)(::False) = True()
 
 Base.all(::Tuple{Vararg{True}}) = true
 Base.all(::Tuple{Vararg{Union{True, False}}}) = false
-Base.all(::Tuple{Vararg{False}}) = false
+Base.all(::Tuple{False, Vararg{False}}) = false
```

Comment:
This will still add definitions for `Base.all(::Tuple{})` etc. since `Vararg`s maybe empty. To avoid this, arguments should be restricted to tuples of length >= 1:
```suggestion
Base.all(::Tuple{True, Vararg{True}}) = true
Base.all(::Tuple{Union{True,False}, Vararg{Union{True, False}}}) = false
```